src/vs/workbench/contrib/chat/browser/chatAccessibilityService.ts

```diff
@@ -17,29 +17,31 @@ export class ChatAccessibilityService extends Disposable implements IChatAccessi
 
 	private _responsePendingAudioCue: IDisposable | undefined;
 	private _hasReceivedRequest: boolean = false;
+	private _runOnceScheduler: RunOnceScheduler | undefined;
 
 	constructor(@IAudioCueService private readonly _audioCueService: IAudioCueService) {
 		super();
 	}
 	acceptRequest(): void {
 		this._audioCueService.playAudioCue(AudioCue.chatRequestSent, true);
-		this._register(disposableTimeout(() => {
+		this._runOnceScheduler = new RunOnceScheduler(() => {
 			if (!this._hasReceivedRequest) {
 				this._responsePendingAudioCue = this._audioCueService.playAudioCueLoop(AudioCue.chatResponsePending, CHAT_RESPONSE_PENDING_AUDIO_CUE_LOOP_MS);
 			}
-		}, CHAT_RESPONSE_PENDING_AUDIO_CUE_LOOP_MS));
+		}, CHAT_RESPONSE_PENDING_AUDIO_CUE_LOOP_MS);
```

Comment:
Sorry, what I meant is to have all of this in the ctor and only call `this._runOnceScheduler.schedule` here. And register the scheduler once as disposable